### DISCUSSION TITLE
How do I add a folder in a Repo?

### DISCUSSION THREAD

**Discussion by opticaldrive** [community/community#61112] [Repositories] [answered]
How do I add a folder in a Repo?

Thank you.

**Comment by jackyyy65** [comment, marked as answer]
Oh, it's quite simple actually! To add a new folder to a repository, I just need to go to the repository on GitHub and click on 'Create new file.' Then, I enter the name I want for the folder, followed by a forward slash (/) to create it. After that, I can choose to add a file inside the folder if I want, or just create an empty folder. Once I'm done, I hit 'Commit new file,' and ta-da! The new folder is now part of the repository. It's such a handy trick to organize my projects neatly on GitHub!

**Reply by opticaldrive** [reply]
Thanks!

**Comment by ghost** [comment]
not working

**Comment by MelanieS** [comment]
This is how I do it... once I figured it out https://youtu.be/IeGElVLEbao
